src/containers/header/header.jsx

```diff
@@ -136,7 +136,7 @@ const Header = ({ isMenuOpen, showMenu, hideMenu, height, user, theme }) => (
             }
         </MenuHeader>
         {isMenuOpen && (
-            <MenuBar>
+            <MenuBar onClick={isPhoneResolution ? hideMenu : null}>
```

Comment:
It'll catch all bubbling `click` events. Also it'll catch click on wrapper (`MenuBar`). If user miss link header will became collapsed.

Reply:
if (window.innerWidth < VALUE) document.getElementById(ID).style.display = 'none'; worked since 2001 (IE6). why use such complicated calls in 2018?

Reply:
we should modify DOM elements manually in react apps

Reply:
kitos, not shure i understood you. as i see here you need to hide/show navigation block. you dont need to modify DOM at all. you just need to set style.display for your navigation block element. one-liner above will suffice (well, it depends on your document flow, of course. in normal flow 'content after navigation' on setting display to 'none' for navigation block content will automatically redraws at proper place)

Reply:
working example for you http://bitofsky.org/kitos.htm

Reply:
>just need to set style.display

it is DOM modification

if you are using react, only react can do that. So react way would be:
`<Menu style={{ display: shouldDisplayMenu ? 'block' : 'none' }}>...`

but in this case we'll spend time on rendering menu block though we do not show it

so our way is
`
{isMenuOpen && <MenuBar>...
`

Reply:
well, i mean style.display does not lead to any DOM structure changes, just reflow and redraw in browser. so it is mostly cheap as not touching DOM at all. am i wrong?
how you can render DOM elements with display set to none? they just sit in DOM not rendered by browser. or we are talking about different 'render'? and after all you still need your menubar to be in DOM to be showed to user by request (show/hide switch). it's better to have it in DOM from initial page load, than constructing and inserting it into DOM on the fly (or maybe i dont understand how you implement it). am i wrong here?

Reply:
by the way, i cant find toggleMenu() function in your repo. can you help me please?